core/jazz_acl/package.json

```diff
@@ -10,6 +10,7 @@
   },
   "dependencies": {
```

Comment:
Can you update  other default values that would have come with the template - name, desc, main etc.?

Reply:
updated

Reply:
updated